Core/Util/GameManager.cpp

```diff
@@ -580,7 +605,7 @@ bool GameManager::InstallZippedISO(struct zip *z, int isoFileIndex, std::string
 		allBytes += zstat.size;
 	}
 
-	std::string outputISOFilename = g_Config.currentDirectory + "/" + fn.substr(nameOffset);
+	std::string outputISOFilename = destination + "/" + fn.substr(nameOffset);
```

Comment:
The previous code is probably for the use case of clicking on a zip on your local device (i.e. downloaded homebrew from Chrome or something.)  This change would probably make that use case more confusing.

-[Unknown]

Reply:
Oh, I didn't know that was possible. I thought only downloads from store were permitted. As I saw it, there are two implementations for the iso/cso. One that just allows for raw isos to be downloaded and the other where the iso is on root of a zip. And for the fileutil function allocation, do you recommend to be made in its own class or do you have something else in mind? 
Thanks